### ISSUE TITLE
bug with plugins that use searchers

### ISSUE THREAD

**Issue by FREYA265** [primary-theme/obsidian#204]
Hi Cecilia, I noticed this bug with the search filters.
Another Primary user also has the same bug.

![image](https://github.com/primary-theme/obsidian/assets/139784879/c0a9b7ca-bbce-4503-bd22-1ad7f5c4a351)

It would seem to come from this line: 

```
.popover, .suggestion-container, .suggestion-popover {
    animation: var(--anim-popup); 
}
```

Thank you for your work :)

**Comment by Signynt** [comment]
Just adding a few more examples of this showing up, and mentioning that depending on the window size the suggestion dialogue sometimes shows up on the bottom left.

QuickAdd:
![Screenshot 2024-07-11 at 14 41 40](https://github.com/user-attachments/assets/7ff006a1-96cc-4e0b-938c-11dc290b829e)

Templater:
![Screenshot 2024-07-11 at 14 41 33](https://github.com/user-attachments/assets/a8eb5955-9131-4c72-9224-7997e1b68d62)

Projects:
![Screenshot 2024-07-11 at 14 38 50](https://github.com/user-attachments/assets/e46c0e98-5c3a-48fe-b076-e3441e198c4e)
![Screenshot 2024-07-11 at 14 38 35](https://github.com/user-attachments/assets/877412df-8627-42d4-8ba3-da8ed627a3c7)

**Comment by ceciliamay** [comment]
finally fixed this one guys! will be available to 2.10.0-beta this weekend 🫡
